tools/hangar/__snapshots__/invalid.ts.snap

```diff
@@ -44,6 +44,20 @@ error: Unknown symbol \\"this\\"
   |     ^^^^ Unknown symbol \\"this\\"
 
 
+error: Unknown symbol \\"this\\"
```

Comment:
What introduced this change in this snapshot?

Reply:
Checking

Reply:
This is actually the correct errors.